app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False)\
+            -> None:
```

Comment:
Use parentheses () instead of backslashes \ to separate lines.
```suggestion
    def __init__(
        self,
        name: str,
        health: int = 100,
        hidden: bool = False
    ) -> None:
```

Reply:
Done